raspberry-pi/4/audio.nix

```
@@ -0,0 +1,46 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.hardware.raspberry-pi."4".audio;
+in
+{
+  options.hardware = {
+    raspberry-pi."4".audio = {
+      enable = lib.mkEnableOption ''
+        Enable configuration for audio
+      '';
+    };
+  };
+
+  config = lib.mkIf cfg.enable {
+    hardware.deviceTree = {
+      overlays = [
+        # Equivalent to dtparam=audio=on
+        {
+          name = "audio-on-overlay";
+          dtsText = ''
+            /dts-v1/;
+            /plugin/;
+            / {
+              compatible = "brcm,bcm2711";
+              fragment@0 {
+                target = <&audio>;
+
+              __overlay__ {
```

```suggestion
                __overlay__ {
```

Indentation is off by one.

---

[I blame you](https://gist.github.com/samueldr/e83102caf431ccef8bc2e825f9273ca7) :P